src/Tests/dotnet-list-package.Tests/GivenDotnetListPackage.cs

```diff
@@ -181,7 +181,7 @@ public void ItListsTransitivePackage()
                 .Should()
                 .Pass()
                 .And.NotHaveStdErr()
-                .And.HaveStdOutContaining("System.IO.FileSystem");
+                .And.NotHaveStdOutContaining("System.IO.FileSystem");
```

Comment:
I think the prior version of newtonsoft had a transitive dependency on filesystem. Rather than changing this to a not check, there is presumably some other system library that you should just for here and in like 176.

Reply:
The following screenshot shows the result of running the command dotnet list pack --include-transitive before and after changing the newtonsoft.json version. From the output, there is no transitive package after the change.

![image](https://user-images.githubusercontent.com/46013274/175883112-9f00b53a-26a4-4c76-a77b-0fd90ed36ab2.png)

Reply:
If that's the case, let's leave this test unchanged then as it's testing version 9 for a reason. We could try to find another package that has a transitive reference but the value of doing so probably wouldn't be worth it.

Reply:
I made a change to make this test stand alone.